### PR TITLE
Normalize Unit Testing workflow across repos

### LINKED ISSUE #1069: Each phpunit test takes about 600 ms
There are about 370 tests and phpunit takes 220 seconds, which equals to about 600 ms per test.

Analyse why the tests are so slow. Is that due webserver based on built-in php webserver?

Each basic atk4 ui response should not take longer than 60 ms.

### LINKED ISSUE #1071: Coverage can not be turned of for local/maual tests
`--no-coverage` option of phpunit is not honored. Some coverage helpers are currently hardcoded in test files.

Important for quick local testing and to be able to run tests without xdebug.

### LINKED ISSUE #1108: Disable wget verbose in tests
Example: https://travis-ci.org/github/atk4/ui/jobs/675692696#L230

Important to be able to compare test outputs quickly.

### PR DESCRIPTION
**Pull request by mvorisek**
- Fixes #1071
- Fixes right image version
- Fixes #1108
- Fixes #1069 - 10x 🚀
- Moves Behat before phpunit on Travis - display results for the frontend gurus first ;-)

Migrating of Behat itself from Travis is not part of this PR.

This PR is 1:1 replacement of the original one without any BC.

Xdebug is disabled if no coverage is required (reuired = "latest" PHP which is send to CodeCov) which is the magic behind 10x faster phpunit.

### CODE REVIEW

**Comment by mvorisek** [comment]
This PR cut the test times of non-latest php version 10x! (from 4 minutes to 25 seconds)

File: composer.json [hunk 1]
```diff
@@ -21,7 +21,13 @@
   "prefer-stable": true,
   "authors": [
     {"name": "Romans Malinovskis", "email": "romans@agiletoolkit.org", "homepage": "https://nearly.guru/" },
-    {"name": "Michael Voříšek", "homepage": "https://mvorisek.cz/"}
+    {"name": "Alain Belair"},
```
Comment (mvorisek) [inline composer.json]: @ibelar Is it ok to add you as a 2nd major contributor and do you have a homepage to add?

File: composer.json [hunk 2]
```diff
@@ -21,7 +21,13 @@
...
+    {"name": "Imants Horsts", "homepage": "https://darkside.lv/"},
+    {"name": "Francesco Danti", "homepage": "https://oracoltech.com/"},
+    {"name": "Michael Voříšek", "homepage": "https://mvorisek.cz/"},
+    {"name": "Philipp Grashoff"},
```
Comment (mvorisek) [inline composer.json:28]: @PhilippGrashoff Is it ok to add you to the authors as a major contributor? Do you have a homepage to add?

File: composer.json [hunk 3]
```diff
@@ -21,7 +21,13 @@
...
+    {"name": "Alex Cicovic", "homepage": "https://alexcicovic.com/"},
+    {"name": "Georgi Hristov"}
```
Comment (mvorisek) [inline composer.json]: @georgehristov Is it ok to add you to the authors as a major contributor? Do you have a homepage to add?

File: composer.json [hunk 4]
```diff
@@ -21,7 +21,13 @@
   "prefer-stable": true,
   "authors": [
     {"name": "Romans Malinovskis", "email": "romans@agiletoolkit.org", "homepage": "https://nearly.guru/" },
-    {"name": "Michael Voříšek", "homepage": "https://mvorisek.cz/"}
+    {"name": "Alain Belair"},
+    {"name": "Imants Horsts", "homepage": "https://darkside.lv/"},
+    {"name": "Francesco Danti", "homepage": "https://oracoltech.com/"},
+    {"name": "Michael Voříšek", "homepage": "https://mvorisek.cz/"},
+    {"name": "Philipp Grashoff"},
+    {"name": "Alex Cicovic", "homepage": "https://alexcicovic.com/"},
```
Comment (mvorisek) [inline composer.json:29]: @acicovic  Is it ok to add you to the authors as a major contributor?

Comment (acicovic) [reply composer.json:29]: Yes, it's fine with me.

File: composer.json (hunk 1 above)
Comment (ibelar) [reply composer.json]: oh - thanks - no home page.

File: composer.json (hunk 3 above)
Comment (georgehristov) [reply composer.json]: Thanks @mvorisek , you can use https://xsystems.io/

File: tests/BuiltInWebServerAbstract.php [hunk 5]
```diff
@@ -38,7 +38,13 @@ public static function setUpBeforeClass(): void
         }
 
         // The command to spin up the server
-        self::$process = Process::fromShellCommandline('php -S ' . self::$host . ':' . self::$port . ' -t ' . self::getPackagePath());
+        $cmdArgs = [
+            '-S', static::$host . ':' . static::$port,
+            '-t', self::getPackagePath(),
+            '-d', 'open_basedir=' . ini_get('open_basedirh'),
```
Comment (georgehristov) [inline tests/BuiltInWebServerAbstract.php]: `open_basedirh` typo?

Comment (mvorisek) [reply tests/BuiltInWebServerAbstract.php]: @georgehristov Perfect catch!

Comment (mvorisek) [reply tests/BuiltInWebServerAbstract.php]: fixed

**Review by DarkSide666 — APPROVED** [review]
Didn't read all changes, but overall looks good to merge.
If there will be some issues later, then @mvorisek can fix them with new (and smaller) PRs.

File: composer.json (hunk 2 above)
Comment (romaninsh) [reply composer.json:28]: Current contributors here: https://github.com/atk4/infrastructure/blob/master/projects/agiletoolkit.org/github/agiletoolkit.org/github.tf#L6

But may not include our past contributors... 

https://github.com/atk4/ui/graphs/contributors

Comment (mvorisek) [reply composer.json:28]: Yes, I used the GH data in each repo and asked each contributor for an approval if they want to be added. All home pages collected except for "Philipp Grashoff". This is closed to me and merged already.

**Pull request merged**